lib/matplotlib/bezier.py

```diff
@@ -478,28 +445,3 @@ def make_wedged_bezier2(bezier2, width, w1=1., wm=0.5, w2=0.):
                                      c3x_right, c3y_right)
 
     return path_left, path_right
-
-
-@cbook.deprecated(
```

Comment:
I would just leave these in place as they'll go away soon enough anyways.  (hint: the smaller your diff, the easier to get a review ;-))

Reply:
Thanks for the tip! Didn't move them on purpose really, just that I wrote this code before Ernest merged #16773, I'll put them back in their places to make the diff (significantly) smaller.